sdk/eidolon_ai_sdk/agent/simple_agent.py

```diff
@@ -98,21 +114,25 @@ class SimpleAgentSpec(BaseModel):
     agent_refs: List[str] = []
     actions: List[ActionDefinition] = [ActionDefinition()]
     cpu: AnnotatedReference[APU] = None
-    cpus: Optional[List[NamedCPU]] = []
+    apu: AnnotatedReference[APU] = None
```

Comment:
Apu should have no default since default is handled in annotated. I'm not sure if this causes errors or not.